packages/react-ui-kit/package.json

```diff
@@ -25,9 +25,11 @@
     "jest": "25.1.0",
     "jest-emotion": "10.0.32",
     "react": "16.13.0",
+    "react-docgen-typescript": "1.16.3",
```

Comment:
Since we don't want to show props from `node_modules` but only our own, do we still need this lib?

Reply:
you're right, removed in https://github.com/wireapp/wire-web-packages/pull/2937/commits/7ccc1ce466f3278a01c918d72f6801e6a5fa53cc